app/models/event.rb

```diff
@@ -153,6 +153,25 @@ def process_rejection(options)
     end
   end
 
+  def process_cancellation
+    return unless scheduled?
```

Comment:
I've to admit that I don't like to change the data here!
What about just displaying this different in the view?
@hennevogel @differentreality ^^